Marlin/src/lcd/language/language_en.h

```diff
@@ -427,10 +427,13 @@ namespace LanguageNarrow_en {
   LSTR MSG_SHAPING_ZETA                   = _UxGT("@ damping");
   LSTR MSG_SHAPING_A_FREQ                 = STR_A _UxGT(" frequency");
   LSTR MSG_SHAPING_B_FREQ                 = STR_B _UxGT(" frequency");
-  LSTR MSG_SHAPING_A_ZETA                 = STR_A _UxGT(" damping ");
-  LSTR MSG_SHAPING_B_ZETA                 = STR_B _UxGT(" damping ");
+  LSTR MSG_SHAPING_C_FREQ                 = STR_B _UxGT(" frequency");
+  LSTR MSG_SHAPING_A_ZETA                 = STR_A _UxGT(" damping");
+  LSTR MSG_SHAPING_B_ZETA                 = STR_B _UxGT(" damping");
+  LSTR MSG_SHAPING_C_ZETA                 = STR_B _UxGT(" damping");
   LSTR MSG_SHAPING_X_ENABLE               = _UxGT("Enable X shaping");
   LSTR MSG_SHAPING_Y_ENABLE               = _UxGT("Enable Y shaping");
+  LSTR MSG_SHAPING_Z_ENABLE               = _UxGT("Enable Y shaping");
```

Comment:
Copy & paste error. This should be “Enable Z shaping”.

Reply:
i dont get why ` MSG_SHAPING_X_ENABLE `/Y/Z is even there, its not used anywhere as far as I can tell. there are several other `MSG_` unused. actually it would be nice to be able to enable/disable input shaping. because having it as part of the firmware requires you perform the calibration to use it.

meaning prints are affected if it isnt tuned.  
in other words, its not like Linear Advance where you can set it to 0 (as far as I know that is the case). setting input shaping settings to 0 isnt the same as disabling it.

Reply:
I also noticed it isn't used, so went ahead and removed it.